lib/api/core/plugins/pluginsManager.js

```diff
@@ -634,8 +634,7 @@ class PluginsManager {
         if (typeof target !== 'function'
           && typeof plugin.object[target] !== 'function')
         {
-          let message = `Unable to configure pipe for event "${event}" with`
-          + ` provided method. "${target}" should be a plugin method name, or a function.`;
+          let message = `Unable to configure pipe for event "${event}" with provided method. "${target}" should be a plugin method name, or a function.`;
```

Comment:
You maybe forget to apply the new wrapping rules :slightly_smiling_face:

Reply:
We didn't wrap the string